CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## 1.15.0
+
+This relase brings some expanded DNS configuration options, beta support for QEMU, recursive volume mounts, and rkt-monitor improvements.
+
+### New features
+- DNS configuration improvements ([#3161](https://github.com/coreos/rkt/pull/3161)):
+    - Respect DNS results from CNI
+    - Add --dns=host mode to bind-mount the host's /etc/resolv.conf
+    - Add --dns=none mode to ignore CNI DNS
+    - Add --hosts-entry (IP=HOSTNAME) to tweak the pod's /etc/hosts
+    - Add --hosts-entry=host to bind-mount the host's /etc/hosts
+- Introduce QEMU support as an alternative KVM hypervisor ([#2952](https://github.com/coreos/rkt/pull/2952))
+- rkt-monitor: bunch of improvements ([#3093](https://github.com/coreos/rkt/pull/3093))
+- add support for recursive volume/mounts ([#2880](https://github.com/coreos/rkt/pull/2880))
+- stage1: allow systemd notifications from the app in the container to the host ([#2826](https://github.com/coreos/rkt/pull/2826))
+
+### Other changes
+- makefile/kvm: add dependency for copied files ([#3197](https://github.com/coreos/rkt/pull/3197))
+- store: refactor GetRemote ([#2975](https://github.com/coreos/rkt/pull/2975)).
+- build,stage1: include systemd dir when checking libs ([#3186](https://github.com/coreos/rkt/pull/3186))
+- tests: volumes: add missing test `volumeMountTestCasesNonRecursive` ([#3165](https://github.com/coreos/rkt/pull/3165))
+- kvm/pod: disable insecure-options=paths for kvm flavor ([#3155](https://github.com/coreos/rkt/pull/3155))
+- stage0: don't copy image annotations to pod manifest RuntimeApp annotations ([#3100](https://github.com/coreos/rkt/pull/3100))
+- stage1: shutdown.service: don't use /dev/console ([#3148](https://github.com/coreos/rkt/pull/3148))
+
+### Documentation & Packaging
+- Documentation: update Ubuntu installation ([#3196](https://github.com/coreos/rkt/pull/3196)).
+- build: build simple .deb and .rpm packages ([#3177](https://github.com/coreos/rkt/pull/3177)). Add a simple script to build .deb and .rpm packages. This is not a substitute for a proper distro-maintained package.
+- Documentation/capabilities-guide: remove --cap-retain/remove mentions ([#3192](https://github.com/coreos/rkt/pull/3192)). Partially fixes #3179 (the documentation part)
+- docs: use absolute url in api-service doc ([#3187](https://github.com/coreos/rkt/pull/3187)). This is breaking links on the website, and this way every time we sync we won't have to remember to fix this for the latest version. We only sync the Documentation folder, so linking to things outside of it breaks.
+host to stage1 if they're not already present after building systemd.
+- Documentation/using-rkt-with-kubernetes: add minikube ([#3185](https://github.com/coreos/rkt/pull/3185)). null
+- docs: fix small typo ([#3182](https://github.com/coreos/rkt/pull/3182))
```

Comment:
If it's small typo - should this be noted in changelog?

Hmmm... Seems that we need an update in https://github.com/coreos/rkt/blob/master/CONTRIBUTING.md#format-of-the-commit-message
Once we have `docs`, other time we have `Documentation/something:`, other time `Documentation:`